### PR TITLE
Add paths to windows metadataFiles

### PR DESCRIPTION
**Pull request by gabriel-samfira**
When creating a new empty base layer on Windows, hcsshim creates a set of empty registry hives in C:\Windows\System32\config. These files are needed in order for the new layer to be considered valid.

Add these locations to the metadataFiles, which excludes them when comparing the diffs of two layers.

This will be used in the snapshot tests on Windows in: https://github.com/containerd/containerd/pull/8043 

Signed-off-by: Gabriel Adrian Samfira <gsamfira@cloudbasesolutions.com>

### CODE REVIEW

**Comment by gabriel-samfira** [comment]
These files get created here: https://github.com/microsoft/hcsshim/pull/1637/files#diff-94af4df272f2d4cf11cff84cc70fe9ac3a715b0e2c13440bb5e131f56660806cR23

**Review by dcantah — APPROVED** [review]

File: fs/fstest/compare_windows.go [hunk 1]
```diff
@@ -19,6 +19,14 @@ package fstest
 // TODO: Any more metadata files generated by Windows layers?
```
Comment (estesp) [inline fs/fstest/compare_windows.go:19]: nit: should this TODO: line be removed now that the additional files are added by this PR?

Comment (dcantah) [reply fs/fstest/compare_windows.go:19]: Could argue there still may be more 😅 but I agree, probably time to get rid of this.

Comment (gabriel-samfira) [reply fs/fstest/compare_windows.go:19]: indeed. There may be more, but we can just add them as we discover them. Removing the TODOs

Comment (gabriel-samfira) [reply fs/fstest/compare_windows.go]: done

**Review by estesp — APPROVED** [review]
LGTM

**Review by austinvazquez — APPROVED** [review]

**Review by kzys — APPROVED** [review]

**Pull request merged**